src/BuildingRegistry.Api.BackOffice.Abstractions/BackOfficeContext.cs

```diff
@@ -31,6 +32,28 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<BuildingUnitBuilding>()
                 .Property(x => x.BuildingPersistentLocalId);
+
+
+            modelBuilder.Entity<BuildingUnitAddressRelation>()
```

Comment:
Don't we typically add this logic to an  IEntityTypeConfiguration<BuildingUniAddressRelation> class?

Reply:
Typically yes, but since it's small, I don't mind.